### PR TITLE
Make ch5_load.sh script more robust

### PR DESCRIPTION
**Pull request by alexott**
multiple changes:
* Automatically detect path to the data
* use DSBulk from PATH if it's there
* Allow to customize destination keyspace/graph
* fix for DSBulk 1.3.4 vs 1.4.0 compatibility issue: `-schema.allowMissingFields`  doesn't
  work with 1.4.0 - we must use `--schema.allowMissingFields`

### CODE REVIEW

**Comment by denisekgosnell** [comment]
Thank you @alexott

File: data/ch5/ch5_load.sh [hunk 1]
```diff
@@ -1,29 +1,46 @@
...
+
+# 
+OLDWD="`pwd`"
+SCRIPTIR="`dirname $0`"
+cd "$SCRIPTDIR"
+DATADIR="`pwd`"
+
+DSBULK="`which dsbulk`"
+if [ -z "$DSBULK" ]; then
+    if [ ! -f "$DSBULK_PATH/bin/dsbulk" ]; then
```
Comment (denisekgosnell) [inline data/ch5/ch5_load.sh:15]: @alexott how do we handle when a user sets the DSBULK_PATH but does not have an environment variable?

eg: my local setup breaks the logic here. I set DSBULK_PATH at the top of this script, but I do not have an environment variable for dsbulk set in my PATH.

File: data/ch5/ch5_load.sh [hunk 2]
```diff
@@ -1,29 +1,46 @@
...
+        echo "Please set DSBULK_PATH variable to top-level path of DSBulk distribution"
+        echo "  or add directory with dsbulk to PATH, like, 'PATH=...dsbulk-1.3.4/bin:\$PATH'"
+        exit 1
+    fi
```
Comment (alexott) [inline data/ch5/ch5_load.sh:19]: We need to add `DSBULK=$DSBULK_PATH/bin/dsbulk` After fi.

File: data/ch5/ch5_load.sh (hunk 1 above)
Comment (alexott) [reply data/ch5/ch5_load.sh:15]: See my comment below. I thought that I’d tested this part, but it looks like I didn’t :-(

File: data/ch5/ch5_load.sh (hunk 2 above)
Comment (denisekgosnell) [reply data/ch5/ch5_load.sh:19]: @alexott thank you! this resolves the issue.

I have cherry-picked this commit to the top of this branch. Then, I refactored all other loading processes to reflect the suggestions. I will open a PR into master for you to review.

Comment (alexott) [reply data/ch5/ch5_load.sh:19]: @denisekgosnell I'll review your PR tomorrow, I have some suggestions for your open PR, similar to the https://github.com/datastax/graph-book/pull/3, but I need to make similar changes in the all scripts. If you can look to my PR, and it looks ok for you, I can make changes in all scripts

**Pull request merged**